src/APIFunctions/S3Bucket.js

```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+
+export function uploadToS3(files) {
+  for (let i = 0; i < files.length; i++) {
+    let file = files[i];
+    let fileParts = files[i].name.split('.');
```

Comment:
is it possible that your backend could just take the filename as a whole (`sce.png`) and then handle the logic for splitting it there?

Reply:
I put it there since I needed the fileType down below at the headers. Shoud I still change it?